packages/animations/lib/src/modal.dart

```diff
@@ -27,6 +29,7 @@ typedef _ModalTransitionBuilder = Widget Function(
 /// modal route that will be displayed, such as the enter and exit
 /// transitions, the duration of the transitions, and modal barrier
 /// properties.
+/// By default, `configuration` is [FadeScaleTransitionConfiguration].
```

Comment:
Ahh, sorry about that, I should have looked at the rest of the file. What you have is fine. The only change I might propose is just making it a part of the previous block of sentences, but it's just a nice-to-have:
```suggestion
/// properties. By default, `configuration` is 
/// [FadeScaleTransitionConfiguration].
```